From c2e4e8f1ccb741fb0e039c0580b211f290814884 Mon Sep 17 00:00:00 2001
From: Carlo Caione <ccaione@baylibre.com>
Date: Wed, 14 Nov 2018 18:45:36 +0000
Subject: [PATCH 32/55] mender: apollo: Import generated
 config_mender_defines.h

This is the configuration generated by mender when the U-Boot
integration is enabled with the following settings:

MENDER_ROOTFS_PART_A = "/dev/mmcblk0p1"
MENDER_ROOTFS_PART_B = "/dev/mmcblk0p2"
BOOTENV_SIZE = "0x2000"
MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET = "0"

Signed-off-by: Carlo Caione <ccaione@baylibre.com>
---
 include/config_mender_defines.h | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 include/config_mender_defines.h

diff --git a/include/config_mender_defines.h b/include/config_mender_defines.h
new file mode 100644
index 0000000000..74180f3e5f
--- /dev/null
+++ b/include/config_mender_defines.h
@@ -0,0 +1,32 @@
+/* AUTOGENERATED FILE - DO NOT EDIT! */
+/* This file is provided by the meta-mender layer. */
+
+#ifndef HEADER_CONFIG_MENDER_DEFINES_H
+#define HEADER_CONFIG_MENDER_DEFINES_H
+
+/* Shell variables */
+#define MENDER_BOOT_PART_NUMBER 1
+#define MENDER_BOOT_PART_NUMBER_HEX 1
+#define MENDER_ROOTFS_PART_A_NUMBER 1
+#define MENDER_ROOTFS_PART_A_NUMBER_HEX 1
+#define MENDER_ROOTFS_PART_B_NUMBER 2
+#define MENDER_ROOTFS_PART_B_NUMBER_HEX 2
+#define MENDER_UBOOT_STORAGE_INTERFACE "mmc"
+#define MENDER_UBOOT_STORAGE_DEVICE 0
+
+/* BB variables. */
+#define MENDER_STORAGE_DEVICE_BASE "/dev/mmcblk0p"
+#define MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_1 0x0
+#define MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_2 0x800000
+#define MENDER_ROOTFS_PART_A_NAME "/dev/mmcblk0p1"
+#define MENDER_ROOTFS_PART_B_NAME "/dev/mmcblk0p2"
+
+/* For sanity checks. */
+#define MENDER_BOOTENV_SIZE 0x2000
+
+#define MENDER_BOOT_KERNEL_TYPE "booti"
+#define MENDER_KERNEL_NAME "Image"
+#define MENDER_DTB_NAME "meson-axg-apollo-evt0.dtb"
+#define MENDER_UBOOT_PRE_SETUP_COMMANDS ""
+#define MENDER_UBOOT_POST_SETUP_COMMANDS ""
+#endif /* !HEADER_CONFIG_MENDER_DEFINES_H */
-- 
2.19.1

